tests/framework/layer/test_layer.cpp

```diff
@@ -266,6 +462,19 @@ FRAMEWORK_EXPORT VKAPI_ATTR VkResult VKAPI_CALL vkEnumerateDeviceExtensionProper
                                                                                      VkExtensionProperties* pProperties) {
     return test_vkEnumerateDeviceExtensionProperties(physicalDevice, pLayerName, pPropertyCount, pProperties);
 }
+FRAMEWORK_EXPORT VKAPI_ATTR VkResult VKAPI_CALL vkEnumeratePhysicalDevices(VkInstance instance, uint32_t* pPhysicalDeviceCount,
+                                                                           VkPhysicalDevice* pPhysicalDevices) {
+    return test_vkEnumeratePhysicalDevices(instance, pPhysicalDeviceCount, pPhysicalDevices);
+}
+FRAMEWORK_EXPORT VKAPI_ATTR void VKAPI_CALL vkGetPhysicalDeviceProperties(VkPhysicalDevice physicalDevice,
+                                                                          VkPhysicalDeviceProperties* pProperties) {
+    return test_vkGetPhysicalDeviceProperties(physicalDevice, pProperties);
+}
+FRAMEWORK_EXPORT VKAPI_ATTR VkResult VKAPI_CALL vkEnumeratePhysicalDeviceGroups(
+    VkInstance instance, uint32_t* pPhysicalDeviceGroupCount, VkPhysicalDeviceGroupProperties* pPhysicalDeviceGroupProperties) {
+    return test_vkEnumeratePhysicalDeviceGroups(instance, pPhysicalDeviceGroupCount, pPhysicalDeviceGroupProperties);
+}
```

Comment:
Why are these exported? I don't think they exist in the group of functions that the loader directly loads.

https://github.com/KhronosGroup/Vulkan-Loader/blob/master/docs/LoaderLayerInterface.md#layer-interface-version-0

Reply:
Because the ones above were.  I was just being consistent.  This is all surrounded by "#if TEST_LAYER_EXPORT_ENUMERATE_FUNCTIONS".